### ISSUE TITLE
Clarify results for numerical PFR in the _ss.out

### ISSUE THREAD

**Issue by wittregr** [VlachosGroup/openmkm#23]
Can you please clarify that for a numerical PFR set of results is each row in the _ss.out files a set of values corresponding to the quantities entering the node? Or are they the quantities leaving that node?

**Comment by MaxRCohen** [comment]
Hi @mbkumar, could we get this clarification?

**Comment by skasiraj** [comment]
From what I can see OpenMKM prints the solution of each node (after integration to Steady State), which presumably would be the values corresponding to the quantities exiting the node. 

The naming of the nodes themselves (using volume) corresponding to the center of the CSTR can be discussed as mentioned in the other issue #22.

**Comment by MaxRCohen** [comment]
Yes, this is an important point worth discussing - it is important that both the math must be correct and our documentation's explanation of the math.
